docs/api/api/api.txt

```diff
@@ -1997,6 +1997,7 @@ XmlResult: status_ok
 
 DELETE /staging/<staging_workflow_project>/staging_projects/<staging_project>/staged_requests
   Remove one or multiple bs_requests to the staged requests of a staging project.
+XmlBody: remove_staged_requests
```

Comment:
Good catch. Curious that this same file is used in line 1995 for the POST request.

Reply:
Yes... I was thinking if we should rename the file, even though it's the same structure for both.